software/cloud-dashboard/react-client/src/containers/bucket-view.js

```diff
@@ -72,6 +72,12 @@ class BucketView extends Component {
             }
         }
 
+    var flowDataValues = [];
+    for (var i in flowData) {
+      flowDataValues[i] = flowData[i].flow;
+    }
+    var yAxisMinMax = [Math.min.apply(Math, flowDataValues), Math.max.apply(Math, flowDataValues)];
```

Comment:
@suyashkumar do you really have to do `Math.min.apply(Math...` to work with an array?!